example/main.go

```diff
@@ -59,6 +60,12 @@ func gen(opts *options) error {
 		SourceDir: opts.source,
 		TargetDir: opts.target,
 		Plugin:    true,
+		ManHeader: &doc.GenManHeader{
+			Title:   "BUILDX",
+			Section: "1",
```

Comment:
Wondering if we will have cases where  not all man-pages must be in the same section https://man7.org/linux/man-pages/man7/man-pages.7.html

e.g. for the cli, we also currently have `Dockerfile (5)` and `dockerd (8)`. I _think_ in most/all cases a binary would only be in a single section, it's possible that, e.g. a man page describing things like `--config` could also be in section 5 ("config")

Reply:
Yes indeed we should take care of it. Will make some changes related to that.